docs/usage.md

```diff
@@ -1,55 +1,120 @@
 
 **Note 1:** Depending on the configuration file used, this process may download multiple-hundreds of gigabytes of data. This may take quite a while. Use the optional `log-level=debug` command line flag for more verbose output to track command execution. 
 
-**Note 2:** After `oc-bundle create full` has finished, an imageset named bundle_000000.tar will have been created and available in your current directory. Use this file with `oc-bundle publish` to mirror the imageset to a disconnected registry.
-
-### Create Diff
-
-Once a full imageset has been created and published, differential imagesets that contain only updated images as per the configuration file can be generated with the following command:
-
-`oc-bundle create diff --dir=< empty directory >> --config=<< path to config file >>`
-
-**Note 1:** Depending on the configuration file used and the periodicity between running `oc-bundle create diff`, this process may download multiple-hundreds of gigabytes of data, though differential updates should usually result in a significantly smaller imagesets.  
-
-**Note 2:** After `oc-bundle create diff` has finished, an imageset named bundle_000000.tar will have been created and available in your current directory. This file must be used in sequence and with a publishing workspace that has a matching UUID.
-
-## Publish
+**Note 2:** After `oc-mirror` has finished, an imageset named mirror_seq1_000000.tar will have been created and available in your specified directory. Use this file with `oc-mirror` to mirror the imageset to a disconnected registry:
 
-During the publishing phase, an imageset is checked against the state of a publishing workspace and then mirrored from file to a target registry server. Full imagesets must be used in conjunction with an empty workspace and differential imagesets must be published in sequence with a workspace that shares the same UUID as the incoming imageset.
+`./bin/oc-mirror --from archives --dir test-publish docker://localhost:5000`
 
-To publish a full imageset use the following command:
+### Running `oc-mirror` For Differential Updates
 
-`oc-bundle publish --dir=<< Empty Directory >> --archive=bundle_000000.tar --to-mirror=<< fqdn:port of target mirror >>`
+Once a full imageset has been created and published, differential imagesets that contain only updated images as per the configuration file can be generated with the same command as above:
 
-To publish a differential imageset use the following command:
+`./bin/oc-mirror --config imageset-config.yaml --dir test-create file://archives`
 
-`oc-bundle publish --dir=<< previously used dir >> --archive=bundle_000000.tar --to-mirror=<< fqdn:port of target mirror >>`
+**Note 1:** Depending on the configuration file used and the periodicity between running `oc-mirror`, this process may download multiple-hundreds of gigabytes of data, though differential updates should usually result in a significantly smaller imagesets.  
```

Comment:
We should probably hoist this note as a warning to the top of this doc, and generalize it to say "this tool might downloaded many gigabytes of data to your local disk. ensure you have ~100GB free space before running the tool".